SPARQLWrapper/Wrapper.py

```diff
@@ -557,70 +566,63 @@ def setHTTPAuth(self, auth):
             valid_types = ", ".join(_allowedAuth)
             raise ValueError("Value should be one of {0}".format(valid_types))
 
-    def setQuery(self, query):
+    def setQuery(self, query: Union[str, bytes]) -> None:
         """
-            Set the SPARQL query text. 
+        Set the SPARQL query text.
 
-            .. note::
-              No check is done on the validity of the query
-              (syntax or otherwise) by this module, except for testing the query type (SELECT,
-              ASK, etc). Syntax and validity checking is done by the SPARQL service itself.
+        .. note::
+          No check is done on the validity of the query
+          (syntax or otherwise) by this module, except for testing the query type (SELECT,
+          ASK, etc). Syntax and validity checking is done by the SPARQL service itself.
 
-            :param query: query text.
-            :type query: string
-            :raises TypeError: If the :attr:`query` parameter is not an unicode-string or utf-8 encoded byte-string.
+        :param query: query text.
+        :type query: string
+        :raises TypeError: If the :attr:`query` parameter is not an unicode-string or utf-8 encoded byte-string.
         """
-        if isinstance(query, str):
-            pass
+        if not isinstance(query, (str, bytes)):
```

Comment:
This looks like it can go into a separate PR.

Reply:
934e334